client/components/status/status.test.js

```diff
@@ -456,6 +456,31 @@ describe("<Status /> interactions", () => {
     await tick();
     expect(spyFn.mock.calls.length).toBe(0);
   });
+  it("should submit loginFormRef after login", async () => {
+    validateToken.mockReturnValue(true);
+    props = createTestProps();
+    props.location.search = "";
+    props.userData = {
+      response_code: "AUTH_TOKEN_VALIDATION_SUCCESSFUL",
+      radius_user_token: "o6AQLY0aQjD3yuihRKLknTn8krcQwuy2Av6MCsFB",
+      username: "tester",
+      email: "tester@tester.com",
+      is_active: true,
+      is_verified: true,
+      phone_number: "+237672279436",
+    };
+    expect(true).toBe(true);
+    sessionStorage.setItem(`${props.orgSlug}_just_authenticated`, true);
+    wrapper = shallow(<Status {...props} />, {
+      context: {setLoading: jest.fn()},
+      disableLifecycleMethods: true,
```

Comment:
I think this line should be removed so the `componentDidMount` method should be called automatically without the need of us calling it here.